### ISSUE TITLE
Items linked with mmoitems Item tag is different

### ISSUE THREAD

**Issue by ninesik** [PluginBugs/Issues-ItemsAdder#2288]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.

### Discord tag (optional)

nine#1927

### What happened?

A bug in which items created with gui through the ia command and items created with the iagive command do not overlap because the tags are different.

### Steps to reproduce the issue

1. /ia
2. take out the Items linked with mmoitems.
3. /iagive name <same item as number 2>
4. do not overlap each other

### Server version

[20:54:14 INFO]: Current: git-Fusion-"2d64c0b" (MC: 1.19.3)*
Previous: git-Fusion-"5bbfcd5" (MC: 1.19.3)
* You are 4 version(s) behind
Download the new version at: https://github.com/IPECTER/Fusion/releases

### ItemsAdder Version

[20:54:56 INFO]: ItemsAdder version 3.3.0b-r5
[20:54:56 INFO]: Customized Minecraft without client mods!
[20:54:56 INFO]: Website: devs.beer
[20:54:56 INFO]: Author: LoneDev

### ProtocolLib Version

[20:55:04 INFO]: ProtocolLib version 5.0.0-SNAPSHOT-b612
[20:55:04 INFO]: Provides read/write access to the Minecraft protocol.
[20:55:04 INFO]: Authors: dmulloy2 and comphenix

### LoneLibs Version

[20:55:17 INFO]: LoneLibs version 1.0.23
[20:55:17 INFO]: Collection of libs used by my plugins
[20:55:17 INFO]: Authors: LoneDev and others

### LightAPI Version (optional)

x

### LibsDisguises Version (optional)

x

**Comment by LoneDev6** [comment]
Are you sure it's not because some stats are randomized?
Try to get it multiple times, you'll see it won't stack anyway because of the random mmoitems stats.

**Comment by ninesik** [comment]
It is not randomized.
{AttributeModifiers:[{Amount:0.0d,AttributeName:"minecraft:generic.attack_speed",Name:"mmoitemsDecoy",Operation:0,UUID:[I;-2021319128,-1357757450,-1987131861,-563359508]}],CustomModelData:10009,HSTRY_ENCHANTS:'{"Stat":"ENCHANTS","OGStory":[{"MMOITEMS_ENCHANTS_ñstr":"[]"}]}',HideFlags:2,MMOITEMS_DYNAMIC_LORE:'["&c소모품"]',MMOITEMS_ENCHANTS:"[]",MMOITEMS_INEDIBLE:1b,MMOITEMS_ITEM_ID:"광물랜덤상자3",MMOITEMS_ITEM_TYPE:"CONSUMABLE",MMOITEMS_NAME:"&f&l상급 광물상자",display:{Lore:['{"italic":false,"color":"red","text":"소모품"}'],Name:'{"bold":true,"italic":false,"color":"white","text":"상급 광물상자"}'},itemsadder:{id:"consumable3",namespace:"inmc"}}


{AttributeModifiers:[{Amount:0.0d,AttributeName:"minecraft:generic.attack_speed",Name:"mmoitemsDecoy",Operation:0,UUID:[I;-2021319128,-1357757450,-1987131861,-563359508]}],CustomModelData:10009,HSTRY_ENCHANTS:'{"Stat":"ENCHANTS","OGStory":[{"MMOITEMS_ENCHANTS_ñstr":"[]"}]}',HideFlags:2,MMOITEMS_DYNAMIC_LORE:'["&c소모품"]',MMOITEMS_ENCHANTS:"[]",MMOITEMS_INEDIBLE:1b,MMOITEMS_ITEM_ID:"광물랜덤상자3",MMOITEMS_ITEM_TYPE:"CONSUMABLE",MMOITEMS_NAME:"&f&l상급 광물상자",display:{Lore:['{"extra":[{"italic":false,"color":"red","text":"소모품"}],"text":""}'],Name:'{"extra":[{"bold":true,"italic":false,"color":"white","text":"상급 광물상자"}],"text":""}'},itemsadder:{id:"consumable3",namespace:"inmc"}}

**Comment by ninesik** [comment]
![image](https://user-images.githubusercontent.com/41807903/214562449-bf0193f3-5e81-44d8-8e81-1f6527a05212.png)

**Comment by LoneDev6** [comment]
I'll do some more tests, it might be an issue of the MMOitems api.
I'll keep you updated.

**Comment by LoneDev6** [comment]
This bug was introduced by the randomized properties compatibility update.
I implemented a fix for that issue which will be available on 3.4.0b.

Video: https://streamable.com/0vm0hz

Config:
```yml
info:
  namespace: test
items:
  test_mmoitems:
    display_name: test_mmoitems
    resource:
      generate: false
      model_path: "item/diamond"
      material: BREAD
    mmoitem:
      id: SANDWITCH
      type: CONSUMABLE 
```
